pyne/mesh.py

```diff
@@ -101,9 +102,15 @@ def __set__(self, mesh, value):
         self[:] = value[:]
 
     def __delete__(self, mesh):
+        del self.mesh.tags[self.name]
```

Comment:
I think we can just get rid of these `__delete__` methods on the tag classes, but they aren't doing any harm.

Reply:
I think it's probably good to keep them in case anyone does ever use this class in a way that invokes those methods.... 
Does it make sense to add the same default for the `mesh` argument? I don't really understand how that might be invoked to know if it could ever be invoked without an argument?

Reply:
I guess in the current version it is required, so I'll leave it required but then be sure to use that `mesh` in the code